CHANGELOG.md

```diff
@@ -25,6 +25,11 @@
   (e.g. `@testable`, `@_exported`, ...).  
   [hiltonc](https://github.com/hiltonc)
 
+* Add new `--quiet-deprecation-warnings` flag that silences deprecation
```

Comment:
https://github.com/realm/SwiftLint/pull/5070

Reply:
Oh bummer! Thank you for the prompt correction.